web_sdk/BUILD.gn

```diff
@@ -587,38 +587,38 @@ if (!is_fuchsia) {
       tmp_files += [
         {
           source = rebase_path(source)
-          destination = rebase_path(source, "$root_build_dir")
+          destination = rebase_path(source, "$root_build_dir/flutter_web_sdk")
         },
       ]
     }
     tmp_files += [
       {
         source = rebase_path("$root_out_dir/canvaskit/canvaskit.js")
-        destination = "flutter_web_sdk/canvaskit/canvaskit.js"
+        destination = "canvaskit/canvaskit.js"
       },
       {
         source = rebase_path("$root_out_dir/canvaskit/canvaskit.wasm")
-        destination = "flutter_web_sdk/canvaskit/canvaskit.wasm"
+        destination = "canvaskit/canvaskit.wasm"
       },
       {
         source = rebase_path("$root_out_dir/canvaskit_chromium/canvaskit.js")
-        destination = "flutter_web_sdk/canvaskit_chromium/canvaskit.js"
+        destination = "canvaskit/chromium/canvaskit.js"
       },
       {
         source = rebase_path("$root_out_dir/canvaskit_chromium/canvaskit.wasm")
-        destination = "flutter_web_sdk/canvaskit_chromium/canvaskit.wasm"
+        destination = "canvaskit/chromium/canvaskit.wasm"
       },
```

Comment:
Thanks for doing the chromium change as well! I guess I'm not gonna need my restructuring PR after all (https://github.com/flutter/engine/pull/39796).

Reply:
Actually though, the copy steps from that PR might still be pertinent. I don't have any copy steps here.